examples/Tutorial_1-carrots_demo/carrots_demo.py

```diff
@@ -0,0 +1,408 @@
+#!/usr/bin/env python
```

Comment:
We already have a file called [carrots.py](https://github.com/OpenMined/PyDP/blob/dev/examples/Tutorial_1-carrots_demo/carrots.py). Is there a reason why this file was added?